### PR TITLE
[Feature][Connector-V2] Support opengauss-cdc

### PR DESCRIPTION
**Pull request by dailai**
### Purpose of this pull request

Link issue #7410 

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [x] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Review by hailin0 — COMMENTED** [review]
please update this file
https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml

**Comment by dailai** [comment]
> https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml

I looked. There's already a cdc

image

File: docs/en/connector-v2/source/Opengauss-CDC.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+# Opengauss CDC
```
Comment (hailin0) [inline docs/en/connector-v2/source/Opengauss-CDC.md:1]: please update zh docs

**Comment by Hisoka-X** [comment]
Looking great! Thanks @dailai

File: docs/en/connector-v2/source/Opengauss-CDC.md (hunk 1 above)
Comment (dailai) [reply docs/en/connector-v2/source/Opengauss-CDC.md:1]: done

**Comment by Carl-Zhou-CN** [comment]
niubility!

**Review by Carl-Zhou-CN — APPROVED** [review]
LGTM

**Review by hailin0 — APPROVED** [review]
LGTM

**Pull request merged**